pydantic/_internal/_generate_schema.py

```diff
@@ -1972,7 +1972,6 @@ def _computed_field_schema(
         return_type_schema = self._apply_field_serializers(
             return_type_schema,
             filter_field_decorator_info_by_field(field_serializers.values(), d.cls_var_name),
-            computed_field=True,
         )
```

Comment:
I believe you can remove the `computed_field` argument from the `_apply_field_serializers` function as well.

https://github.com/pydantic/pydantic/blob/a6dc87285f93f90c2d5c298ee7c52f5d7e878194/pydantic/_internal/_generate_schema.py#L2196C9-L2201

Reply:
Otherwise, lgtm! Thanks!

Reply:
https://github.com/pydantic/pydantic/pull/10390/files#diff-2ebfa892ec1d09440eefd4b22f488ec38c1cde365394e5a4cea6edb484827ba5R2198-R2199
@sydney-runkle I've already removed it in this commit.

Reply:
Oh great! Not sure how I missed that 🤨 thanks!